pgcli/pgclirc

```diff
@@ -31,6 +31,12 @@ multi_line_mode = psql
 # "off" - skip all warnings
 destructive_warning = all
 
+# The list of commands when checking for a destructive query.
+# Note: "update" is based on the destructive_warning level.
+# Here are few more commands you may want to add to this list:
+# insert, create, grant
+destructive_commands = drop, shutdown, delete, truncate, alter, update 
```

Comment:
You probably could have refactored the old `destructive_warning` setting, without adding a new one. Instead of warning level, it would now explicitly contain a list of destructive commands.

To support existing users of pgcli, you could do one of the following:

1. rewrite their configs to replace:

| old                     | new                              |
|-------------------------|----------------------------------|
| destructive_warning=all | destructive_warning=drop, shutdown, delete, truncate, alter, update |
| destructive_warning=moderate | destructive_warning=drop, shutdown, delete, truncate, alter |
| destructive_warning=off | destructive_warning=off          |

2. or, detect old values (`all`, `moderate` or `off`) and replace accordingly in the code.

TBH, I don't know how safe rewriting is with configobj. I remember we had an issue of merged configs losing comments, so there's that. You'd have to do some digging.

Reply:
Reusing it makes sense. I'll get that changed.